### PR TITLE
250 Leadership section: adjust padding on desktop and change wording

### LINKED ISSUE #250: Leadership section padding updates
Frame 847

### PR DESCRIPTION
**Pull request by chrisnottca**
## Reminder

> Please ensure the following criteria is met for this PR. It will help others review your PR and provide confidence things work as expected.

- [x] CI is green
- [x] Link to any related issues
- [ ] Received at least 1 approval from core members
- [ ] Made sure that changes are obvious for the reviewer (ex meaningful title and commit messages, comments in code or PR where appropriate, screenshots, etc.)

# What Changed?

- Adjust top and bottom padding on desktop
- Change "Executive" to "Leadership"

> Related Issue: https://github.com/commitdev/commit.dev/issues/250

- TODO

## Screenshots

Screen Shot 2021-04-13 at 10 49 10 AM

### CODE REVIEW

File: sections/executive-team-section/index.js [hunk 1]
```diff
@@ -14,10 +14,10 @@ import { people } from './people'
 const { h2 } = Heading.VARIANT
 
 const StyledSection = styled.section`
-  padding: ${rem('100px')} 0;
+  padding: ${rem(140)} 0 ${rem(140 - 84)};
```
Comment (ALukonin) [inline sections/executive-team-section/index.js]: Hey Chris, `rem` does accept strings or numbers, but would it be better to keep consistency among the file/project?

Comment (chrisnottca) [reply sections/executive-team-section/index.js]: That's a really good question. I guess one benefit to using numbers is the ability to do calculations. But consistency is important as well so we need to decide if we should switch and, if so, how we go about it (change a file if we are working on it for other reasons or as a single dedicated task).

Comment (alebiavati) [reply sections/executive-team-section/index.js]: I think this is something we can address in a different "consistency" issue

Comment (alebiavati) [reply sections/executive-team-section/index.js]: @chrisnottca what does the 84 mean? apart from the consistency issue, I think we should try to not use "magic" numbers. If there are specific numbers like this, let's make variables to improve code readability.

**Review by alebiavati — CHANGES_REQUESTED** [review]

**Review by alebiavati — APPROVED** [review]

**Pull request merged**